### PR TITLE
Miscellaneous fixes

### PR DESCRIPTION
**Pull request by juanlofer-eprosima**

### CODE REVIEW

File: docs/CMakeLists.txt [hunk 1]
```diff
@@ -53,3 +53,9 @@ compile_documentation()
 compile_test_documentation(
     "${PROJECT_SOURCE_DIR}/test" # Test directory
 )
+
```
Comment (jparisu) [inline docs/CMakeLists.txt:56]: I am not sure we should install this project when there is nothing installed from it.
I would fix the warning by checking if DOCS_BUILD (or whatever it is called) is set, and if not, do nothing so the project does not get compiled (or even `project` call is not done)

**Review by jparisu — CHANGES_REQUESTED** [review]
Add echo values to yaml validator (if we stop maintaining that tool we should deprecate it properly, for now these changes are very simple to do so lest do them)

File: docs/CMakeLists.txt (hunk 1 above)
Comment (juanlofer-eprosima) [reply docs/CMakeLists.txt:56]: That was my first attempt, couldn't make it work.

**Review by jparisu — APPROVED** [review]
LGTM

**Pull request merged**